code/modules/clothing/spacesuits/rig/suits/riot.dm

```diff
@@ -1,7 +1,7 @@
 /obj/item/weapon/rig/riot
 	name = "riot rig"
 	desc = "A heavy duty and armoured rig suit, designed for riot control and shipboard disciplinary enforcement. This suit is commonly found among security officers on the Sprawl, and in lesser capacity on private ships and stations."
-	icon_state = "ds_police_rig"
```

Comment:
what's going on here? we had the black riot rig sprites specially made, and we're still discussing the uses of new ones

Reply:
You really didn't read any of the summary post that explains this, did you?